v0.7/mlperf.conf

```diff
@@ -4,11 +4,13 @@
 # All times are in milli seconds
 
 # set performance_sample_count for each model
-mobilenet.*.performance_sample_count_override = 1024
 resnet50.*.performance_sample_count_override = 1024
 ssd-mobilenet.*.performance_sample_count_override = 256
 ssd-resnet34.*.performance_sample_count_override = 64
-gnmt.*.performance_sample_count_override = 3903900
+bert.*.performance_sample_count_override = 10833
+dlrm.*.performance_sample_count_override = 204800
+rnnt.*.performance_sample_count_override = 2513
```

Comment:
This is the total number of sequences we get by removing >15sec sequences from dev-clean set. @galv Could you check if this matches your number? Thanks

Reply:
2513 is right.